ui/src/app/machines/views/MachineDetails/MachinePCIDevices/MachinePCIDevices.test.tsx

```diff
@@ -5,22 +5,16 @@ import configureStore from "redux-mock-store";
 
 import MachinePCIDevices from "./MachinePCIDevices";
```

Comment:
The diff is a bit hard to read - might be easier to just look directly at the files for `MachinePCIDevices.tsx`, `MachineUSBDevices.tsx` and `NodeDevices.tsx` and see if the api and separation makes sense.